### ISSUE TITLE
Mitochondria annotations for kasthuri11cc dataset

### ISSUE THREAD

**Issue by rmarren1** [openconnectome/ndio-demos#3]
Having trouble getting the mitochondria truth data.

As in the notebook I use

``` python
mito_anno = oo.get_cutout('kasthuri2015_ramon_v1', 'mitochondria', 694, 1794, 1750, 2640, 1004, 1154, resolution=3) # for annotations
mito_img = oo.get_cutout('kasthuri11cc', 'image', 694, 1794, 1750, 2640, 1004, 1154, resolution=3) # for images
```

mito_img works great, but mito_anno returns just 0'd out numpy arrays:

``` python
>>> mito_anno = oo.get_cutout("kasthuri2015_ramon_v1", "mitochondria", 694, 1794, 1750, 2640, 1004, 1154, resolution =3)
>>> mito_anno
array([[[0, 0, 0, ..., 0, 0, 0],
        [0, 0, 0, ..., 0, 0, 0],
        [0, 0, 0, ..., 0, 0, 0],
        ..., 
        [0, 0, 0, ..., 0, 0, 0],
        [0, 0, 0, ..., 0, 0, 0],
        [0, 0, 0, ..., 0, 0, 0]]], dtype=uint32)
```

Thanks!

**Comment by j6k4m8** [comment]
This is a great question — and also a misleading result of how numpy prints matrices! 

If you run some analysis on `mito_anno`, you'll see that it's not really all zeros; there's actually quite a bit of data hidden in _thick 0 padding_.

You can see this padding super clearly in an image like this, taken from the 'bottom' of your matrix:
![](http://openconnecto.me/ocp/ca/kasthuri2015_ramon_v1/mitochondria/xy/3/694,1794/1750,2640/1154/)

As you can see, the actual annotations are super sparse. There's white, or `0`, all around it.

To confirm this, try running the following:

``` python
>>> len(mito_anno[mito_anno != 0])
1097155
```

 In other words, there are `1097155` cells in this matrix that are nonzero, which sounds about right.

If you try running this again and you can confirm you're really getting all zeros, let me know and I'll take another look!

**Comment by rmarren1** [comment]
Ah thank you! I didn't realize the first few pictures were unannotated (guessing they are to be the testing data).
